toolchain/sem_ir/file.h

```diff
@@ -148,16 +148,17 @@ class File : public Printable<File> {
     return name_scopes_[scope_id.index];
   }
 
-  // Adds a node to a specified block, returning an ID to reference the node.
-  auto AddNode(NodeBlockId block_id, Node node) -> NodeId {
+  // Adds a node to the node list, returning an ID to reference the node.
+  auto AddNodeNoBlock(Node node) -> NodeId {
```

Comment:
Why "NoBlock" when there's no variant that adds it to a block?

Reply:
There is a variant that adds the node to a block in `Check::Context` rather than `SemIR::File`. I think calling both functions `AddNode` would be error-prone -- it seems easy to accidentally use `context.semantics_ir().AddNode` instead of `context.AddNode`. Given that this function is the rare / "strange" operation, I thought it best to call the `Check::Context` one `AddNode` and make this name more verbose to draw attention to it doing something unusual.

Extended the function comment to point the reader at the Check::Context::AddNode function.

Reply:
Renamed to `AddNodeInNoBlock`.